src/cascadia/TerminalSettingsEditor/Profiles.xaml

```diff
@@ -26,251 +29,145 @@ the MIT License. See LICENSE in the project root for license information. -->
-                        </StackPanel>
-                    </Grid>
-                </ScrollViewer>
-            </PivotItem>
-            <PivotItem x:Uid="Profile_Advanced">
-                <Grid Background="{ThemeResource ApplicationPageBackgroundThemeBrush}"
-          Margin="0,12,0,0">
-                    <Grid.RowDefinitions>
-                        <RowDefinition Height="Auto" />
-                        <RowDefinition Height="*" />
-                    </Grid.RowDefinitions>
-                    <Grid.ColumnDefinitions>
-                        <ColumnDefinition Width="1*" />
-                        <ColumnDefinition Width="1*" />
-                    </Grid.ColumnDefinitions>
-                    <StackPanel Grid.Row="1" Grid.Column="0" Margin="0,0,100,0">
-                        <CheckBox x:Uid="Profile_Hidden" IsChecked="{x:Bind State.Profile.Hidden, Mode=TwoWay}"/>
-                        <CheckBox x:Uid="Profile_SuppressApplicationTitle" IsChecked="{x:Bind State.Profile.SuppressApplicationTitle, Mode=TwoWay}"/>
-                        <Controls:RadioButtons x:Uid="Profile_AntialiasingMode"
+                    <StackPanel Orientation="Horizontal">
+                        <TextBox x:Uid="Profile_BackgroundImage"
+                                 x:Name="BackgroundImage"
+                                 Text="{x:Bind State.Profile.BackgroundImagePath, Mode=TwoWay}"/>
+                        <Button x:Uid="Profile_BackgroundImageBrowse"
+                                Click="BackgroundImage_Click"
+                                Margin="10,0,0,0"/>
+                    </StackPanel>
+                    <Controls:RadioButtons x:Uid="Profile_BackgroundImageStretchMode"
+                                           ItemsSource="{x:Bind BackgroundImageStretchModeList, Mode=OneWay}"
+                                           SelectedItem="{x:Bind CurrentBackgroundImageStretchMode, Mode=TwoWay}"
+                                           ItemTemplate="{StaticResource EnumRadioButtonTemplate}"/>
+                    <ComboBox x:Uid="Profile_BackgroundImageAlignment">
+                        <ComboBox.Items>
+                            <ComboBoxItem x:Uid="Profile_BackgroundImageAlignmentCenter"/>
+                            <ComboBoxItem x:Uid="Profile_BackgroundImageAlignmentLeft"/>
+                            <ComboBoxItem x:Uid="Profile_BackgroundImageAlignmentTop"/>
+                            <ComboBoxItem x:Uid="Profile_BackgroundImageAlignmentRight"/>
+                            <ComboBoxItem x:Uid="Profile_BackgroundImageAlignmentBottom"/>
+                            <ComboBoxItem x:Uid="Profile_BackgroundImageAlignmentTopLeft"/>
+                            <ComboBoxItem x:Uid="Profile_BackgroundImageAlignmentTopRight"/>
+                            <ComboBoxItem x:Uid="Profile_BackgroundImageAlignmentBottomLeft"/>
+                            <ComboBoxItem x:Uid="Profile_BackgroundImageAlignmentBottomRight"/>

```

Comment:
not doing a 9-grid of buttons today?

Reply:
Pushing that one off since it'll need a converter.